api/api.proto

```diff
@@ -937,6 +937,28 @@ service OpenStorageCluster {
     }
 }
 
+// OpenStorageClusterPair service provides the methods to manage a cluster pair
+service OpenStorageClusterPair{
+  // Creates Pair with a remote cluster and returns details about the remote cluster
+  //
+  // ##### Example
+  // {% codetabs name="Golang", type="go" -%}
+  // id, err := client.Create(context.Background(), &api.SdkVolumeCreateRequest{
```

Comment:
The comment is using SdkVolumeCreateRequest, it should be SdkClusterPairCreateRequest.